winterfell/webapp/src/js/claimConfirmModifyContactCtrl.js

```diff
@@ -0,0 +1,29 @@
+var app = angular.module('vetafiApp');
+app.controller('claimConfirmModifyContactCtrl', ['$scope', '$uibModalInstance',
+  function($scope, $uibModalInstance) {
+    var modalData = $scope.modalData;
+
+    $scope.forType = modalData.forType;
+    $scope.targetIndex = modalData.targetIndex;
+    $scope.modalTitle = '';
+    if (modalData.forType == 'address') {
+      $scope.modalTitle = $scope.targetIndex >= 0 ? 'Edit Address' : 'Add Address';
+    } else if (modalData.forType == 'email') {
+      $scope.modalTitle = $scope.targetIndex >= 0 ? 'Edit Email' : 'Add Email';
+    }
+
+    $scope.onCloseModal = function() {
+      $uibModalInstance.close();
```

Comment:
You dont need to make a special function for this, the modal child $scope has certain methods prebound to it by the anguar UI plugin, such that in your modal template you can just say `ng-onclick=close()`